### PR TITLE
Added C# example of sending and receiving mail messages with file att…

### PR DESCRIPTION
**Pull request by Sunnivran**
…achments.

### CODE REVIEW

**Comment by Ernestynian** [comment]
_seems like english is the way to go here_

**In my opinion** before we pull this, you should cut out all unnecesary files (add to .gitignore) like all binaries but that precompiled library, leaving only basic directory tree, solution file, some configs if needed and .cs you wrote yourself.

**Comment by Sunnivran** [comment]
I copy that. I already fixed it.

**Comment by Sunnivran** [comment]
Still waiting...

**Comment by Sunnivran** [comment]
Still waiting for your review...

**Comment by Esgariot** [comment]
> Still waiting for your review...

I'm so sorry, not only was I busy last couple of days and had 0 time to do anything university related but also I just found out that my only comment was stuck on "Pending" and I didn't apply it...

File: Examples/CSharpMailExample/CSharpMailExample/CSharpMailExample.csproj [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="OpenPop">
+      <HintPath>..\..\OpenPop.NET 2.0.5\binaries\signed (strong name)\OpenPop.dll</HintPath>
```
[the remark this answers is not in the document]
Comment (Sunnivran) [reply Examples/CSharpMailExample/CSharpMailExample/CSharpMailExample.csproj]: Thanks, I have fixed that.

**Pull request merged**